.github/workflows/release.yml

```diff
@@ -0,0 +1,29 @@
+name: Release Charts
+
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
```

Comment:
`actions/checkout@v2` was realeased in the meantime. The main difference is that it only does a shallow clone. I'm not sure if the update is worth it (and if we would need to unshallow waht v2 does for the releaser to work).

Reply:
will try out, but should work without any problems